rca/static_src/javascript/programmes/components/ProgrammesCategories/CategoriesTablist.scss

```diff
@@ -0,0 +1,40 @@
+.categories-tablist {
+    @include z-index(above-gridlines);
+    position: relative;
+    padding: $gutter;
+    padding-bottom: $gutter * 0.5;
+    // Allow the tablist to overflow into the next column, as much as the contact block below.
+    margin-right: $image-offset-right;
+
+    @include media-query(medium) {
+        padding-bottom: $gutter;
+    }
+
+    &__heading {
+        color: $color--dark-grey;
+        margin-bottom: $gutter * 0.5;
+
+        @include media-query(medium) {
+            margin-bottom: $gutter * 0.75;
+        }
+    }
+
+    &__tab {
+        color: $color--dark-grey;
+        font-family: $font--primary;
+        outline-width: initial;
```

Comment:
I think it’s quite important for these to have focus outlines so I’ve re-added it here.